src/main/java/com/iota/iri/storage/Persistable.java

```diff
@@ -46,5 +47,21 @@
      * @return <tt>true</tt> if we should add the object to a list for an existing key. <tt>false</tt> if we should
      * false if we should replace the old value
      */
-    boolean merge();
```

Comment:
I believe the javadocs here can also be updated. Now `canMerge` means two things:
1. We want to merge data on the same persistence provider
2. We want to merge data from another persistance provider

It happens so that when we want (1) we also want (2).
Anyhows the javadocs only pertain to (1).

docs are also important :-)

Reply:
Yeah the doc addition surprised me a bit. I will edit it.

Reply:
However, for the docs: for my idea is that more generally says if two persistables of the same type can merge with each other.

"Specifies whether two objects of the same type can be merged and their merged result used as a single object. For storing in a persistence provider this means we can append items to an index and for retrieving this means we can merge the results of multiple storage providers"